site/docs/spec.md

```diff
@@ -646,6 +647,25 @@ Each partition field in the fields list is stored as an object. See the table fo
 
 In some cases partition specs are stored using only the field list instead of the object format that includes the spec ID, like the deprecated `partition-spec` field in table metadata. The object format should be used unless otherwise noted in this spec.
 
+#### Partition Field ID Handling
+
+A partition field ID is an integer used to identify a partition field.
+These IDs should not conflict with the IDs required for the other fields in data files.
+To avoid that, iceberg assigns partition field IDs starting at 1000.
```

Comment:
The requirement is that IDs are unique and do not conflict with field IDs for columns in manifest files. The _reference implementation_ assigns partition field IDs starting at 1,000.

Reply:
Updated.